Components/Sources/Components/Components/Shared/Buttons/WKMenuButtonDelegate.swift

```diff
@@ -5,10 +5,12 @@ public protocol WKMenuButtonDelegate: AnyObject {
     func wkMenuButtonDidTap(_ sender: WKMenuButton)
 
 	func wkSwiftUIMenuButtonUserDidTap(configuration: WKMenuButton.Configuration, item: WKMenuButton.MenuItem?)
+    func wkSwiftUIMenuButtonUserDidTapAccessibility(configuration: WKMenuButton.Configuration, item: WKMenuButton.MenuItem?)
 }
 
 public extension WKMenuButtonDelegate {
 	func wkMenuButton(_ sender: WKMenuButton, didTapMenuItem item: WKMenuButton.MenuItem) {}
 	func wkMenuButtonDidTap(_ sender: WKMenuButton) {}
 	func wkSwiftUIMenuButtonUserDidTap(configuration: WKMenuButton.Configuration, item: WKMenuButton.MenuItem?) {}
+    func wkSwiftUIMenuButtonUserDidTapAccessibility(configuration: WKMenuButton.Configuration, item: WKMenuButton.MenuItem?) {}
```

Comment:
Same note as elsewhere about finding a way to remove the need for this extra delegate method here and above.